pkg/build/controller/strategy/util.go

```diff
@@ -148,7 +148,6 @@ func mountVolume(pod *corev1.Pod, container *corev1.Container, objName, mountPat
 			break
 		}
 	}
-	mode := int32(0600)
```

Comment:
We should probably change this to the default of 0644 and leave it at that.

@bparees I found this commit set the mode to 0600 - do you recall why?
https://github.com/openshift/openshift-controller-manager/commit/b4cee01a40fdba76d3aa90dbb773da6ccf433258

Reply:
i don't recall why we did it specifically, but logically it minimizes who can read the mounted secret and we know there's no reason for any additional `group` or `other` permissions to be granted.  Given these are secrets we're mounting, keeping them as secure as possible seems like the right permission setup.

(it doesn't matter *that* much since the only process running in the pod is our process which has access to the content, but hypothetically it removes some additional exposure avenues)

Reply:
i'll also note that it looks like this commit actually went in back in august 2017 (the github ui makes it look like it was 2019).  So it's been this way for a looooong time.

Reply:
full history on this can be seen here:
https://github.com/openshift/origin/pull/15248

tldr it wasn't a change from 644 to 0600, it was net new code that replaced existing code that used 0600.  The commit was part of a larger PR.